core/src/main/resources/reference.conf

```diff
@@ -184,6 +184,8 @@ backend {
         #concurrent-job-limit = 5
 
         run-in-background = true
+        # Configures any logic which should run after the command completes
```

Comment:
ToL: How about `A single line shell command to run after the execution of every command block`?